File: inst/expt/anqi-bug7.R
library(cvxr)
x <- Variable(3)
P <- diag(3)
obj <- MatrixFrac(x, P)
prob <- Problem(Minimize(obj), list(x == c(1,5,3)))

# base::trace("Solver.get_problem_data", tracer = browser, exit = browser, signature = c("Solver"))
# debug(cvxr:::.lin_matrix)
data <- get_problem_data(prob, "SCS")
result <- solve(prob, solver = "SCS", verbose=TRUE)

File: inst/expt/anqi-bug7.py
from cvxpy import *
import canonInterface
import numpy as np

x = Variable(3)
P = np.eye(3)
obj = matrix_frac(x, P)
constraint = [ x == np.array([1, 5, 3]) ]
prob = Problem(Minimize(obj), constraint)
data = prob.get_problem_data(SCS)
prob.solve(SCS, verbose = True)

##result = prob.solve(SCS, verbose = True)
